src/main/resources/application.conf

```diff
@@ -1140,7 +1140,7 @@ opencomputers {
         "minecraft:fire"
     ]
 
-    # Whether to not show your special thinger (if you have one you know it).
```

Comment:
This is, in fact, the official description for the thing described: The Special Thinger™. You mustn't change this.